app/assets/stylesheets/modules/filters.css.scss

```diff
@@ -9,7 +9,7 @@
   width: $filters-width;
 }
 
-.filters__form {
+.filters__div {
```

Comment:
bad name, this should reflect the role of the element on the website and not the used html tag :) without looking at the neighboring structure I'd guess this should be `filters__container`. it can also stay `filters__form` if the perceived role is that of a form.